packages/meta/collection.yaml

```diff
@@ -0,0 +1,26 @@
+packages:
+- category: "meta"
+  name: "cos-minimal"
+  version: "0.6.1"
+  requires:
+  - category: toolchain
+    name: yip
+    version: ">=0"
+  - category: utils
+    name: installer
+    version: ">=0"
+  - category: system
+    name: cos-setup
+    version: ">=0"
+  - category: system
+    name: immutable-rootfs
+    version: ">=0"
+  - category: system
+    name: grub-config
+    version: ">=0"
+  - category: system
+    name: cloud-config
+    version: ">=0"
+  - category: toolchain
```

Comment:
I gave a thought at this. It's better that the meta explictly requests luet, so it is installing in the system the version from our repository. When building a derivative then you have a transient luet version, but at the end you pick up the one from the toolchain by default. 

The user it's still free to opt-out as they can install the packages singularly as it is right now